src/components/PdfViewer/PdfViewer.tsx

```diff
@@ -92,6 +91,16 @@ export function PdfViewer() {
     };
   }, [rootElement, eventBus, linkService, findController, pdfDocument, setViewer]);
 
+  useEffect(() => {
+    if (!rootElement || !viewer) return;
+
+    if (lightThemeEnabled) {
+      rootElement.querySelector(".pdfViewer")?.classList.add("light-theme");
+    } else {
+      rootElement.querySelector(".pdfViewer")?.classList.remove("light-theme");
+    }
```

Comment:
```suggestion
    if (lightThemeEnabled) {
      rootElement.querySelector(".pdfViewer")?.classList?.add("light-theme");
    } else {
      rootElement.querySelector(".pdfViewer")?.classList?.remove("light-theme");
    }
```
If the element might not be there then the `classList` might not be there as well. I wonder why this isn't caught during build.

Reply:
that's because if the element isn't there the rest of the property lookup is aborted. the extra `?` is redundant. It would make sense if it were possible that the element is there but classList isn't.